src/lib.rs

```diff
@@ -800,23 +801,27 @@ impl<T: Counter> Histogram<T> {
     ///
     /// println!("{:?}", hist.iter_percentiles(1).collect::<Vec<_>>());
     ///
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(0.01), 0.01, 1, 1)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(0.01),
+    ///     percentile: 0.01, count_at_value: 1, count_since_last_iteration: 1 }));
```

Comment:
Yes, that's the style I was proposing, though I think `new` would be even better. I recognize that the full `struct` constructor conveys more information for documentation purposes, though I think the version using `new` will be more readable, and possibly more understandable.